test/e2e/testing-manifests/storage-csi/external-snapshotter/rbac.yaml

```diff
@@ -41,6 +41,9 @@ rules:
   - apiGroups: ["snapshot.storage.k8s.io"]
     resources: ["volumesnapshotcontents"]
```

Comment:
Can a lot of this rbac be cleaned up since the sidecar is supposed to have signficantly fewer permissions?

Reply:
Sure.

Reply:
Please don't forget to update https://github.com/kubernetes-csi/external-snapshotter/blob/master/deploy/kubernetes/rbac.yaml

Reply:
It will be updated when the split controller PR is merged.